src/common/regions/types.ts

```diff
@@ -22,6 +22,15 @@ const fipsToPrincipalCityRenames: FipsToPrincipalCityName = {
   [NY_METRO_FIPS]: 'New York City',
 };
 
+// PODT Region object to aid serialization/deserialization w/ JSON
+export interface RegionObject {
+  n: string;
+  u: string;
+  f: FipsCode;
+  p: number;
+  r: RegionType;
```

Comment:
for interface types I prefer to be explicit, because for humans it can be harder to infer types than it is for typescript : )

Reply:
Sorry, what I meant to say is that we don't actually need to write the RegionType field to the JSON file since all of the regions in a given file will be of the same type (state, county, metro).  In fact, your code already doesn't use this field so I think it's just a simple matter of removing it from `RegionObject` interface and the `toObject()` methods.  Not sure if it'll save anything post-gzip, but it is probably cleaner not to write unnecessary data regardless.  😁